deploy/kubernetes/metricbeat-kubernetes.yaml

```diff
@@ -77,6 +77,11 @@ data:
       period: 10s
       host: ${NODE_NAME}
       hosts: ["localhost:10255"]
+      # If using Red Hat OpenShift uncomment this:
+      #hosts: ["https://${HOSTNAME}:10250"]
```

Comment:
@jsoriano Do user also need to remove line 79, or will line 81 overwrite the previous setting if users forget to remove 79?

Reply:
I think that actually this is overwritten, but not sure if this is always the case. It'd be better to mention that previous hosts entry should be removed.